app/controllers/consents_controller.rb

```diff
@@ -12,4 +26,12 @@ def set_session
         params.fetch(:session_id) { params.fetch(:id) }
       )
   end
+
+  def set_patient_sessions
+    @patient_sessions =
+      @session
+        .patient_sessions
+        .includes(patient: :consents)
+        .order("patients.first_name", "patients.last_name")
+  end
```

Comment:
Just a thought ... why is this a `before_action`? It's not being reused, and it's not even using `params` to find a resource, so is it just because we're pulling a resource from the db? Is that enough to qualify it to be a `before_action`? 🤔 

Anyway, not a blocker, it just makes me wish there was a firmer rationale for creating `before_action` hooks.

Reply:
Ah, it's just the pattern we use in every other controller because sometimes we reuse these. Though in this case I don't think the other actions will have a need for the full list of `@patient_sessions`.

I do like separating the `@assignments =` that fetch from the DB with this pattern though. It's like once you get some actions doing it (because it makes sense, for example, `set_session`), it's kind of nice to conform all this type of logic to private methods.